public/app/exercise/exercise.controller.js

```diff
@@ -212,6 +221,8 @@
       exercise.id = data.id;
       exercise.name = data.id.split('.').pop();
 
+      loadEditorContent();
```

Comment:
This line can be removed, since the value of the editor is overriden a few lines later by [`exercise.code = data.code.trim();`](https://github.com/Narxem/webPLM/blob/master/public/app/exercise/exercise.controller.js#L231).
The editor is kept up to date with the variable `exercise.code` using directive from AngularJS [`ng-model = exercise.code`](https://github.com/BuggleInc/webPLM/blob/master/public/app/components/exercise/ide.directive.html#L1)